ocs_ci/ocs/platform_nodes.py

```diff
@@ -126,10 +129,10 @@ def create_nodes(self, node_conf, node_type, num_nodes):
             f'{self.platform.upper()}{self.deployment_type.upper()}Node'
         ]
 
-        rhel_cnt = len(get_typed_worker_nodes('rhel'))
-        rhcos_cnt = len(get_typed_worker_nodes('rhcos'))
+        rhel_count = len(get_typed_worker_nodes('rhel'))
+        rhcos_count = len(get_typed_worker_nodes('rhcos'))
         for i in range(num_nodes):
-            node_id = rhel_cnt + rhcos_cnt + i
+            node_id = rhel_count + rhcos_count + i
```

Comment:
Can we use `get_worker_stacks` to determine what workers already exist for the cluster instead of using the counts here? Like we talked about before if a node is removed it could potentially throw this calculation off. If we use the existing stacks we can simply increment our `id` until we hit one that doesn't exist already in cloudformation.

Reply:
done, introduced a new function to find free slots.